prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.ts

```diff
@@ -142,4 +144,9 @@ export class EnrolleeReviewComponent {
     event?.preventDefault();
     this.route.emit(routePath);
   }
+
+  public isPaperEnrollee(enrollee: Enrollee): boolean {
+    return (enrollee?.gpid?.startsWith(PAPER_ENROLLEE_GPID_PREFIX));
+  }
```

Comment:
Similar code exists in `moh-prime\prime-angular-frontend\src\app\modules\adjudication\pages\enrollee-overview\enrollee-overview.component.ts`, can you refactor to shared location?